src/files/import.js

```diff
@@ -7,39 +7,34 @@ async function uploadFileToLinear(file, issueId) {
   if (ENABLE_DETAILED_LOGGING) {
     console.log(chalk.magenta("Input file object:"), file);
     console.log(chalk.magenta("Type of file:"), typeof file);
-    console.log(chalk.magenta("Is File?"), file instanceof File);
-    console.log(chalk.magenta("Is Blob?"), file instanceof Blob);
   }
 
   let fileName = file.name;
   let fileType = file.type;
   let fileSize = file.size;
 
-  if (!(file instanceof File) && !(file instanceof Blob)) {
```

Comment:
`File` and `Blob` don't exist in Node.js. Also, the only place that calls the `uploadFileToLinear` always passes an object